code/modules/mob/living/emote.dm

```diff
@@ -151,6 +151,12 @@
 	emote_type = EMOTE_AUDIBLE
 	stat_allowed = UNCONSCIOUS
 
+/datum/emote/living/gekker//Planning on adding audio and a race check for felinids at a later date but for now this'll do

```

Comment:
i'm pretty sure this is spelled "gecker", and besides that it can probably go in its own PR

Reply:
If I'd been meaning the noises baby monkeys make, aye it'd be spelt that way. 
But it's meant to be the noises foxes make, if you check Renaults emotes you'll see that he has a chance of gekkering too, spelt much the same way.
I'll get shot of this and make it it's own PR instead.